source/CIS/cis-2.rst

```diff
@@ -809,3 +813,9 @@ Smart contract version 1 removes the need for the contract name, which is why :r
 
 In :ref:`CIS1 the token amount<CIS-1-TokenAmount>` is fixed to u64 which was deemed sufficient for most token smart contracts.
 However, to improve the interoperability with decentralized applications with support for other blockchains using 256-bit integers, the variable-length encoding was introduced making :ref:`CIS2 token amount<CIS-2-TokenAmount>` more flexible.
+
+
+Additions after finalization
```

Comment:
I'm not sure what to name this section.
(Also the specification has a final field in the top, which I'm not sure if should be changed/blocks this edit)

Reply:
I think the current state of the PR is fine